greymatter/modules/fabric/variables.tf

```diff
@@ -1,31 +1,50 @@
-variable "aws_region" {}
+variable "cluster_name" {
+  default = "gm-cluster"
+  description = "Name of the Grey Matter ECS Cluster"
+}
+
+variable "vpc_id" {
+  type = string
+  description = "ID for the VPC of the Grey Matter ECS Cluster"
+}
+
+variable "aws_region" {
+  description = "AWS Region"
+}
 
 variable "execution_role_arn" {
   type        = string
   description = "ecsTaskExecutionRole ARN"
 }
 
 variable "docker_secret_arn" {
-  description = "ARN of AWS secret containing dockerß repository credentials"
+  description = "ARN of existing Secrets Manager secret containing docker credentials (see README)"
 }
 
 variable "service_role_arn" {
   description = "ecsServiceRole ARN"
```

Comment:
`string` is the default value type in TF. So let's either omit it entirely from this file's variable declarations, or provide it on every declaration.